css/index.css

```diff
@@ -143,4 +176,133 @@ footer nav {
 footer nav a {
     color: white;
     text-decoration: none;
```

Comment:
add a margin-right : 30px to the footer nav a here to space out the a tags in your footer nav